peachjam/views/documents.py

```diff
@@ -9,6 +11,48 @@
 from peachjam.registry import registry
 
 
+class RedirectResolver:
+
+    RESOLVER_MAPPINGS = {
+        "africanlii": {"country_code": "aa", "domain": "agp.africanlii.org"},
+        "ghalii": {"country_code": "gh", "domain": "new.ghalii.org"},
+        "lawlibrary": {"country_code": "za", "domain": "lawlibrary.org.za"},
+        "lesotholii": {"country_code": "ls", "domain": "new.lesotholii.org"},
+        "malawilii": {"country_code": "mw", "domain": "malawilii.org"},
+        "namiblii": {"country_code": "na", "domain": "namiblii.org"},
+        "senlii": {"country_code": "sn", "domain": "senlii.org"},
+        "seylii": {"country_code": "sc", "domain": "new.seylii.org"},
+        "sierralii": {"country_code": "sl", "domain": "new.sierralii.org"},
+        "tanzlii": {"country_code": "tz", "domain": "new.tanzlii.org"},
+        "tcilii": {"country_code": "tc", "domain": "new.tcilii.org"},
+        "ulii": {"country_code": "ug", "domain": "new.ulii.org"},
+        "zambialii": {"country_code": "zm", "domain": "new.zambialii.org"},
+        "zanzibarlii": {"place_code": "tz-znz", "domain": "new.zanzibarlii.org"},
```

Comment:
```suggestion
        "zanzibarlii": {"place_code": "tz-znz", "domain": "zanzibarlii.org"},
```

Reply:
@longhotsummer  as we change the domains we will need to update the resolver mappings